### PR TITLE
Update PolyaxonLogger to handle v1 and v0

### LINKED ISSUE #1119: Polyaxon logger internal update
## 🚀 Feature

A cross-reposting of https://github.com/polyaxon/polyaxon/issues/787

### PR DESCRIPTION
**Pull request by mouradmourafiq**
Fixes #1119

Description:

This is a WIP PR to make the Ignite logger compatible with both Polyaxon v0 and v1.

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by vfdev-5** [comment]
@mouradmourafiq thanks for the updates! 
Can you either enable github actions on your ignite fork or run black as described here : https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#formatting-without-pre-commit

**Comment by mouradmourafiq** [comment]
@vfdev-5 I tried the linting options initially, and I got more files changed other than the ones that I modified. I will read the contribution guide, thanks 👍

**Comment by vfdev-5** [comment]
@mouradmourafiq yes, it should be related to the versions we are using for code formatting, it is explained in th guide. Otherwise, just enable github actions on you fork and click on "Update branch" button here and it should auto fix and commit the correctly formatted code.

File: tests/ignite/contrib/handlers/test_polyaxon_logger.py [hunk 1]
```diff
@@ -301,5 +301,8 @@ def no_site_packages():
 
 def test_no_polyaxon_client(no_site_packages):
 
-    with pytest.raises(RuntimeError, match=r"This contrib module requires polyaxon-client to be installed"):
+    with pytest.raises(
+        RuntimeError,
+        match=r"This contrib module requires polyaxon (polyaxon-client if you are using polyaxon v0.x) to be installed",
```
Comment (vfdev-5) [inline tests/ignite/contrib/handlers/test_polyaxon_logger.py:306]:
```suggestion
        match=r"This contrib module requires polyaxon",
```

**Review by vfdev-5 — APPROVED** [review]

**Pull request merged**